### PR TITLE
Stage

### LINKED ISSUE #62: [bug]: Login and Signup is not working
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current behavior

facing problem with login and signup 
Screenshot 2024-02-18 162911
Screenshot 2024-02-18 163032

### Steps to reproduce

1.Verify the functionality of the login and signup processes.
2.Test the login and signup functionalities.

### Environment

None

### Version

Local

### Repository

InfraBoard

### PR DESCRIPTION
**Pull request by aryasoni98**

### CODE REVIEW

**Comment by aryasoni98** [comment]
@Divesh-Suryavanshi Check the file and resolve the conflict.

File: src/pages/signupPage.tsx [hunk 1]
```diff
@@ -71,46 +71,46 @@ const SignUp = () => {
   const [error, setError] = useState('')
   const { setAuthStatus } = useAuth()
 
-  const urlParams = new URLSearchParams(window.location.search)
-  const userId = urlParams.get('userId')
-  const secret = urlParams.get('secret')
-  const verifyParams = {
-    userId: userId || '',
-    secret: secret || '',
-  }
```
Comment (Divesh-Suryavanshi) [inline src/pages/signupPage.tsx:80]: all of these changes highlighted in red color can be safely discarded, as these are the parts of errorneous code causing the problem during signUp process

File: src/pages/signupPage.tsx [hunk 2]
```diff
@@ -71,46 +71,46 @@ const SignUp = () => {
...
-            router.push('/')
-          })
-          .catch((err) => {
-            console.log(err)
-            setError(err.message)
-          })
-        router.push('/')
-      } catch (error: any) {
-        console.log(error)
-        setError(error.message)
```
Comment (Divesh-Suryavanshi) [inline src/pages/signupPage.tsx:108]: remove this too

File: src/pages/signupPage.tsx [hunk 3]
```diff
@@ -71,46 +71,46 @@ const SignUp = () => {
...
+      //     console.log(err)
+      //     setError(err.message)
+      //   })
+      router.push('/')
+    } catch (error: any) {
+      console.log(error)
+      setError(error.message)
     }
-  } else {
-    console.error('userId and/or secret not found or invalid.')
```
Comment (Divesh-Suryavanshi) [inline src/pages/signupPage.tsx:112]: remove it too

**Review by Divesh-Suryavanshi — COMMENTED** [review]
all of these changes highlighted in red color can be safely discarded, as these are the parts of errorneous code causing the problem during signUp process

File: src/pages/signupPage.tsx (hunk 3 above)
Comment (Divesh-Suryavanshi) [inline src/pages/signupPage.tsx:112]: remove it

**Review by Divesh-Suryavanshi — COMMENTED** [review]
all of these changes highlighted in red color can be safely discarded, as these are the parts of errorneous code causing the problem during signUp process

File: src/pages/signupPage.tsx (hunk 1 above)
Comment (Divesh-Suryavanshi) [inline src/pages/signupPage.tsx:80]: remove it

**Review by Divesh-Suryavanshi — COMMENTED** [review]
looks good changes can be merged successfully

**Comment by aryasoni98** [comment]
> looks good changes can be merged successfully

looks good but why code conflict?

**Comment by Divesh-Suryavanshi** [comment]
How am i supposed to make changes in this pull request, pls guide me, do i need to make another pull request targeted on master that closes this?, or there is another way to make changes...

**Comment by aryasoni98** [comment]
> How am i supposed to make changes in this pull request, pls guide me, do i need to make another pull request targeted on master that closes this?, or there is another way to make changes...



> How am i supposed to make changes in this pull request, pls guide me, do i need to make another pull request targeted on master that closes this?, or there is another way to make changes...

@Divesh-Suryavanshi Simple update the master branch then fetch the stage branch update the codebase & remove the conflicts and push the stage branch.

**Pull request merged**